README.md

```diff
@@ -145,3 +145,23 @@ GO111MODULE="on" go get sigs.k8s.io/kind@v0.7.0
 ```
 
 Will start a kind cluster and run the end-to-end tests in `./e2e`.  To remove the cluster after running the tests, pass the `--rm` argument to the script, or run `kind delete cluster --name k8s-image-puller-e2e`.
+
+## Scratch Images
+
+Normally, the image puller cannot pull scratch images, as they do not contain a `sleep` command.
+
+But as of [2021-05-06](https://github.com/che-incubator/kubernetes-image-puller/commit/662f9817d0240043616531d3a2b180a5423c726d), the [latest](https://github.com/che-incubator/kubernetes-image-puller/commit/e28a7fb2b246d43ffb48c8f6fe35090f60c3902c) image puller builds contain a golang-based `sleep` binary that can be copied to your scratch image so that it can then be pulled.
+
+See [this example](https://github.com/eclipse-che/che-machine-exec/commit/62632f753636b5b5ec19ef31ab1928679b193097) showing how to use the sleep command in your container:
+
+```
+FROM quay.io/eclipse/kubernetes-image-puller:e28a7fb as k8s-image-puller
+...
+COPY --from=k8s-image-puller /bin/sleep /bin/sleep
```

Comment:
More of an OpenShift/Kubernetes guts kind of question: If I have an image `quay.io/amisevsk/my-image:v1.0.0` that doesn't include sleep, and I build `quay.io/amisevsk/my-image:v1.0.0-sleep` like in
```dockerfile
FROM quay.io/eclipse/kubernetes-image-puller:e28a7fb as k8s-image-puller
FROM quay.io/amisevsk/my-image:v1.0.0
COPY --from=k8s-image-puller /bin/sleep /bin/sleep
```

can I use `:v1.0.0-sleep` to cache `:v1.0.0` on clusters without having to build the sleep binary into all my releases? In my head that's how layering should work, but I'm not 100% on it.

Reply:
that might work, yeah. need a layer/cache expert. Maybe @davidfestal  knows?

Reply:
yes you can use a child image to cache a parent image, you'll have the layer but not the tag so it will still require to pull tag alias but it will be quite fast

We could build images on the fly to cache them on a cluster or maybe mount the sleep utility, I don't know if it could be possible so we could not touch images at all

Reply:
should it be also documented somehow in the official docs? https://www.eclipse.org/che/docs/che-7/administration-guide/defining-the-list-of-images-to-pull/